### ISSUE TITLE
Scholarly retrieves 20 co-authors at maximum

### ISSUE THREAD

**Issue by AlessandroSpallina** [scholarly-python-package/scholarly#316]
Searching for an author and requesting for the co-authors list with scholarly.fill returns only the first 20 co-authors, even if the author have more co-authors. 
This behaviour isn't good and should be fixed, otherwise using the library to fetch co-author will return a partial information.

Code to replicate the issue:
```
from scholarly import scholarly

def main():
    search_query = scholarly.search_author('Carlos Lima Azevedo')
    author = next(search_query)
    author_dict = scholarly.fill(author, sections=['basics', 'indices', 'coauthors'])
    scholarly.pprint(author_dict)

if __name__ == '__main__':
    main()
```

Looking the output of that code you'll see only 20 co-authors, but in the scholar page of [that author](https://scholar.google.it/citations?user=jQkz9EwAAAAJ&hl=en) there are 40+!

**Comment by arunkannawadi** [comment]
This is relatively solvable if the first three or so authors have a Google Scholar profile, but is almost impossible otherwise to fetch that info from Google Scholar itself. I have a similar issue and working on a workaround, and hope to have a PR soon.

**Comment by arunkannawadi** [comment]
I tested the changes in PR #322 with the scholar in your example and was able to fetch all 44 coauthors from this profile.

**Comment by arunkannawadi** [comment]
Fixed in v1.4
